File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/AfpTpoUpGrunnlag.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;
import java.util.Date;


/**
 * Denne klassen inneholder informasjon om TP-ordningers uførepensjonsgrunnlag. Dette er manuelt registrerte data og ikke hentet fra TP-registeret eller andre eksterne kilder.
 * 
 * @author Frederik Rønnevig (Decisive) - PK-35111, PKKAMEL-4656
 */
 
 public class AfpTpoUpGrunnlag implements Serializable {
 
    /**
     * Beløp som skal benyttes i AFP-beregning dersom bruker har hatt uførepensjon fra TPO
     */
	private int belop;
 
 
	/**
	* Dato som beløpet ovenfor var gyldig
	*/
	private Date virkFom;


	
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/AfpTpoUpGrunnlag.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable
import java.util.*
import java.time.LocalDate

class AfpTpoUpGrunnlag : Serializable {
    /**
     * Beløp som skal benyttes i AFP-beregning dersom bruker har hatt uførepensjon fra TPO
     */
    var belop = 0

    /**
     * Dato som beløpet ovenfor var gyldig
     */
    var virkFomLd: LocalDate? = null

}